src/compiler/moduleSpecifiers.ts

```diff
@@ -179,26 +169,24 @@ namespace ts.moduleSpecifiers {
     }
 
     export function forEachFileNameOfModule<T>(
-        files: readonly SourceFile[],
         importingFileName: string,
         importedFileName: string,
-        getCanonicalFileName: GetCanonicalFileName,
         host: ModuleSpecifierResolutionHost,
-        redirectTargetsMap: RedirectTargetsMap,
         preferSymlinks: boolean,
         cb: (fileName: string) => T | undefined
     ): T | undefined {
-        const redirects = redirectTargetsMap.get(importedFileName);
-        const importedFileNames = redirects ? [...redirects, importedFileName] : [importedFileName];
-        const cwd = host.getCurrentDirectory ? host.getCurrentDirectory() : "";
+        const getCanonicalFileName = hostGetCanonicalFileName(host);
+        const cwd = host.getCurrentDirectory();
+        const redirects = host.redirectTargetsMap.get(toPath(importedFileName, cwd, getCanonicalFileName)) || emptyArray;
+        const importedFileNames = [importedFileName, ...redirects];

```

Comment:
What’s the significance of changing `importedFileName` to go at the beginning instead of the end of the array?

Reply:
There is no significance.. I don't know why we had it other way around but it doesn't change any of the tests we have